spec/requests/vendor_api/1.1/get_multiple_applications_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Vendor API - GET /api/v1.1/applications', type: :request do
+  include VendorAPISpecHelpers
+  include CourseOptionHelpers
+
+  before do
+    stub_const('VendorAPI::VERSION', '1.1')
+  end
+
+  it 'returns applications with interviews' do
+    create_list(
+      :submitted_application_choice,
+      2,
+      :with_completed_application_form,
+      course_option: course_option_for_provider(provider: currently_authenticated_provider),
+      status: 'awaiting_provider_decision',
+    )
+
+    get_api_request "/api/v1.1/applications?since=#{CGI.escape(1.day.ago.iso8601)}"
+    interviews_per_application = parsed_response['data'].map do |application|
+      application['attributes']['interviews']
+    end
+
+    expect(interviews_per_application).to eq([[], []])
```

Comment:
do some interviews need to be setup in order for this test to add any value?

Reply:
It's not pretty looking at `[[], []]` but the presence of empty arrays proves interview sections are included in `v1.1`. In `v1.0` output is `[nil, nil]`. Not sure if returning concrete interview examples adds anything, as interview representation is tested in the presenter specs and the single application request spec.